source/drivers/php.txt

```diff
@@ -17,44 +17,56 @@ MongoDB PHP Driver
 Introduction
 ------------
 
-The currently maintained driver is the `mongodb
-<https://pecl.php.net/package/mongodb>`_ extension available from PECL.
+The PHP driver consists of two components, the MongoDB `extension <https://github.com/mongodb/mongo-php-driver>`_
+and `library <https://docs.mongodb.com/php-library/current/>`_.
```

Comment:
An alternative thought came to me while looking at the PHP table changes below. Rather than link to the documentation for each project here, would it make sense to link to the GitHub repositories themselves (as you do in the tables)? If so, we could then change "API Reference" in the bullet list below to "Documentation". That also may make more sense given that the extension doesn't have a deep link _just_ to API reference, and we need to link to the doc homepage anyway. For the library, I don't think we're losing much by linking to the homepage instead of the reference link directly, since the latter is easily accessible from the main menu.

Reply:
I like linking to their github repos. I'm hesitant to change it from API to documentation because it doesn't follow the format of the other pages. We'd like to move to a standard template for driver pages, while leaving enough room for driver specifics.

Reply:
In this case, should "library" link to https://github.com/mongodb/mongo-php-library then, so both links are consistent?

Reply:
actually I think this is fine looking at it